sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/src/SchemaRegistryAvroSerializer.cs

```diff
@@ -118,12 +147,20 @@ private enum SupportedType
             bool async,
             CancellationToken cancellationToken)
         {
+            if (_groupName == null)
```

Comment:
What if it's empty or whitespace?   It doesn't appear that we're doing any validation for those in the constructor.

Reply:
We'd just send it on to the service and it wouldn't work I presume.